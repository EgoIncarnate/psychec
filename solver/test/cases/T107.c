
static void
do_encode ()
{
  char inbuf[ENC_BLOCKSIZE];
  size_t sum;
  fread (inbuf + sum, ENC_BLOCKSIZE - sum);
}
