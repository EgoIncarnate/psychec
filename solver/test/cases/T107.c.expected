int ENC_BLOCKSIZE ;
uintptr_t fread (uintptr_t,int) ;
